docs/sql/reference/create_table.txt

```diff
@@ -193,6 +193,70 @@ to 1000 milliseconds.
 
 For further details see :ref:`refresh_data` or :ref:`sql_ref_refresh`.
 
+blocks
```

Comment:
blocks.read_only
~~~~~~~~~~~~~

Make a table read-only.

value: Boolean value that defines whether the table is read-only or not. ``true`` allows the have a read-only table. ``write`` allows write operations on and changing of table settings.